src/containers/shared/components/Transaction/AMM/AMMVote/parser.tsx

```diff
@@ -0,0 +1,11 @@
+import { getAMMAccountID } from '../../../../MetaParser'
+
+export function parser(tx: any, meta: any) {
+  const tradingFee = tx.TradingFee
+  const accountID = getAMMAccountID(meta)
+
+  return {
+    tradingFee,
```

Comment:
This transaction also has `Asset` and `Asset2`,

Reply:
I'm not seeing those in the figma files: https://www.figma.com/file/RUfMjaArFHMdnoB2I1KwRX/XPPL.org-Explorer-(AMM)?node-id=2%3A32168&t=fcNd6nbomQvLUsD1-0

Is this a miss?

Reply:
cc @khancode

Reply:
Caleb's right. AMMVote has Asset & Asset2. Refer to my [xrpl.js PR](https://github.com/XRPLF/xrpl.js/pull/2071) for latest transaction fields

Reply:
added.